code/index.html

```diff
@@ -2,21 +2,130 @@
 <html>
   <head>
     <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=7">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Project 2: The News Site</title>
     <link rel="stylesheet" href="./style.css" />
     <!-- dont forget the meta tag for responsiveness -->
   </head>
   <body>
-    <header>
       <!-- Some kind of logo and navbar -->
-    </header>
+    <header>
+      <nav>
+        <ul>
+          <li><a href="https://www.goodreads.com/work/quotes/3078186-the-hitchhiker-s-guide-to-the-galaxy"> News</a></li>
+          <li><a href="https://www.goodreads.com/work/quotes/3078186-the-hitchhiker-s-guide-to-the-galaxy">Contact</a></li>
+          <li><a href="https://www.goodreads.com/work/quotes/3078186-the-hitchhiker-s-guide-to-the-galaxy">About</a></li>
+      </ul>
+    </nav>
+    <h1 class="site-name">STRAWBERRY NEWS <br> 
+     <span> Tastiest news of the week</span></h1>
+    
+    <img src="https://media.istockphoto.com/photos/strawberry-picture-id483079803?b=1&k=20&m=483079803&s=170667a&w=0&h=7zG9ttrpHETUZYihlF0hhI0kzPRoBfHCWKoq6zSrP7E=" 
+    class="logo" 
+    alt="logo"> 
+  </header>
 
-    <section class="big-news">
-      <!-- The big news content -->
-    </section>
+    <!-- The big news content -->
+  <section class="big-news">
+   <main>
+     <h1 class="main-heading">LATEST NEWS</h1>
+      
+<div class="main-content-container">
+    <img src="https://images.unsplash.com/photo-1589533610925-1cffc309ebaa?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxzZWFyY2h8Nnx8c3RyYXdiZXJyaWVzfGVufDB8MHwwfHw%3D&auto=format&fit=crop&w=500&q=60" 
+         alt="main content image"  
+        class="main-content-img">
 
-    <section class="">
+<div class="text">
+     <h1 class="article-heading"> Dolphins clame- "smartest beeings on Earth"</h1>
+      
+     <p class="article"> For instance, on the planet Earth, man had always assumed that he was more intelligent 
+          than dolphins because he had achieved so much—the wheel, New York, wars and so on—whilst all the dolphins 
+          had ever done was muck about in the water having a good time. 
+          But conversely, the dolphins had always believed that they were far more intelligent than man—for precisely the same reasons. 
+          <a href="https://www.goodreads.com/work/quotes/3078186-the-hitchhiker-s-guide-to-the-galaxy"> <span class="read-more">Read more</span></a></p>   
+</div>         
+</div>
+  </main>
+  </section>
+
+  <section>
       <!-- The other news  -->
+   <aside>
+        <div class="related-news-h1-container">
+        <h1 class="related-news">RELATED NEWS</h1>
+      </div>
+
+        <div class="news-grid">
```

Comment:
This section below is very easy to follow - great job with the spacing and indentation here!